src/main/kotlin/io/easybreezy/project/application/project/queryobject/Project.kt

```diff
@@ -0,0 +1,108 @@
+package io.easybreezy.project.application.project.queryobject
+
+import io.easybreezy.infrastructure.query.ContinuousList
+import io.easybreezy.infrastructure.query.PagingParameters
+import io.easybreezy.infrastructure.query.QueryObject
+import io.easybreezy.infrastructure.query.toContinuousList
+import io.easybreezy.infrastructure.serialization.UUIDSerializer
+import io.easybreezy.project.model.Projects
+import io.easybreezy.project.model.team.Roles
+import io.easybreezy.project.model.team.Teams
+import kotlinx.serialization.Serializable
+import org.jetbrains.exposed.sql.JoinType
+import org.jetbrains.exposed.sql.ResultRow
+import org.jetbrains.exposed.sql.select
+import org.jetbrains.exposed.sql.selectAll
+import org.jetbrains.exposed.sql.transactions.transaction
+import java.util.*
+
+class ProjectQO(private val slug: String) : QueryObject<Project> {
+    override suspend fun getData(): Project {
+        return transaction {
```

Comment:
We have to use our TransactionManager instead of exposed `transaction {}` in all places

Reply:
Here you can remove transaction {} at ll, because of it have already called in QueryExecutor